### PR TITLE
Iframe with 'about:blank' src can not send 'postMessage'(close #1026)

### LINKED ISSUE #1026: Iframe with `about:blank` src can not send `postMessage`
Sroll to add:

Uncaught DOMException: Failed to execute 'postMessage' on 'Window': Invalid target origin '' in a call to 'postMessage'.

### PR DESCRIPTION
**Pull request by churkin**
/cc @LavrovArtem @miherlosev

### CODE REVIEW

**Comment by churkin** [comment]
FPR

File: test/client/fixtures/sandbox/event/message-test.js [hunk 1]
```diff
@@ -343,6 +343,23 @@ asyncTest('timeout (added to DOM iframe)', function () {
 
 module('regression');
 
+asyncTest('send message from iframe with "about:blank" src (GH-1026)', function () {
+    var iframe = document.createElement('iframe');
+
+    iframe.id = 'test-' + Date.now;
+    setProperty(iframe, 'src', 'javascript:\'<html><body><script>window.parent.postMessage("gh1026", "*")' +
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/event/message-test.js]: move src value to a separate variable

File: test/client/fixtures/sandbox/event/message-test.js [hunk 2]
```diff
@@ -343,6 +343,23 @@ asyncTest('timeout (added to DOM iframe)', function () {
...
+                               '</sc' + 'ript></body></html>\'');
+    setProperty(window, 'onmessage', function (e) {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/event/message-test.js]: After assertion, need to subscribe `message` handler from window

**Comment by churkin** [comment]
@testcafe-build-bot /retest

**Comment by miherlosev** [comment]
@testcafe-build-bot /retest

**Pull request merged**